content/spin/v2/key-value-store-tutorial.md

```diff
@@ -9,6 +9,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/key-value-
 - [Key Value Store With Spin Applications](#key-value-store-with-spin-applications)
 - [Tutorial Prerequisites](#tutorial-prerequisites)
 - [Creating a New Spin Application](#creating-a-new-spin-application)
+- [Prerequisite](#prerequisite)
```

Comment:
This is very contextless in the TOC.  Maybe it would be better not to use an actual heading for this.